### PR TITLE
Fixes for new dashboard texts

### LINKED ISSUE #2391: Crop description to max 2 lines in the map cards
screen shot 2015-02-20 at 13 03 11

### LINKED ISSUE #2427: Misspelling - “There are not datasets. No datasets no fun” → Proposed text: “You haven’t liked any datesets yet

### LINKED ISSUE #2433: Copy-edit - remove the “to”: “We recommend you export your dataset before deleting.”

### PR DESCRIPTION
**Pull request by viddo**
- Fixes #2391 by [truncating text more:](https://github.com/CartoDB/cartodb/issues/2391#issuecomment-75962937)

description with a larger characters: screen shot 2015-02-25 at 16 14 38 
description with a smaller characters: screen shot 2015-02-25 at 16 15 33

---

- Fixes #2433 show export encouragement -text for delete datasets dialog: screen shot 2015-02-25 at 15 44 16

---

- Fixes #2427 replaced “_There are not datasets. No datasets no fun_” with proposed text screen shot 2015-02-25 at 16 07 43

### CODE REVIEW

**Comment by viddo** [comment]
@saleiva @xavijam please review.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1925/))

**Comment by xavijam** [comment]
Only texts? In that case, not much to say, :+1:.

File: lib/assets/javascripts/cartodb/new_dashboard/maps/maps_item.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ var LikesView = require('../../new_common/views/likes/view');
 var Utils = require('cdb.Utils');
 cdb.admin = require('cdb.admin');
 
-var SHORT_DES = 85;
+var SHORT_DES = 45;
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/new_dashboard/maps/maps_item.js]: No css solution?

**Comment by saleiva** [comment]
:+1:

**Comment by xavijam** [comment]
IMHO 45 characters for map description is too short.

**Comment by viddo** [comment]
Yes, it's short, and as I wrote on the related issue it's sub-optimal but there is no perfect solution. The CSS approach works perfect on webkit, but for the rest there would be no ellipsis (or other indicator of more text). If you're fine with the following scenario I'm happy to change it to a CSS solution:
- webkit: screen shot 2015-02-25 at 14 52 00
- non-webkit: screen shot 2015-02-25 at 14 51 49

**Comment by xavijam** [comment]
How about to adding a white gradient at the end?
For the rest of the browsers, i mean. cc @piensaenpixel

**Comment by piensaenpixel** [comment]
+1 @xavijam love your comment

**Comment by viddo** [comment]
Ah yes, you're right that should do. For the sake of simplicity I would leave the gradient since it's so subtle anyway. Looks OK?

screen shot 2015-02-25 at 16 56 52
screen shot 2015-02-25 at 16 57 11

**Comment by viddo** [comment]
retest this please

**Comment by piensaenpixel** [comment]
@viddo  one question, what do you think about remove the gradient for non-webkit browsers? really if we can show the ellipsis, i think we dont need the gradient

**Comment by viddo** [comment]
But the ellipsis is only available for webkit, so I suggest to leave the gradient.

**Comment by piensaenpixel** [comment]
Sorry... it was my fault. I mean remove the gradient for webkit browsers ;)

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1929/))

**Comment by piensaenpixel** [comment]
and one more thing did you test this when you selected the map? you should change the color of the gradient. :)

**Comment by viddo** [comment]
@piensaenpixel: Fixed the hover/selected states (good catch!) in the last commit ffb1855.
However I do not see any issue way to opt-out of the gradient for webkit only. That's why I suggested to leave it, to not complicate things by special rules and hacks.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1932/))

**Comment by viddo** [comment]
@xavijam fixed privacy indicator in FF as well as the map card titles that was not ellipsed properly there either, see last two commits.

**Comment by xavijam** [comment]
:us:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1937/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1949/))

**Comment by piensaenpixel** [comment]
:us:

**Pull request merged**